trigger/version.go

```diff
@@ -120,17 +120,13 @@ func (t *versionTriggers) makeData(sub *event.Subscription, pastTenseOverride st
 		}
 
 		// Look at collective status because we don't know whether the last patch to finish in the version was a child or a parent.
-		patchStatus, err := p.CollectiveStatus()
+		versionStatus, err = p.CollectiveStatus()
```

Comment:
nit: I would still call this `patchStatus` since it's for the patch document and not the version.

Reply:
Personally I'd rather not, since this is being used for a version trigger.